inference-engine/src/gna_plugin/frontend/layer_quantizer.hpp

```diff
@@ -483,6 +483,13 @@ class DataQuantizer<Desc, InferenceEngine::CNNLayer *> : public DataQuantizerBas
                     outData->setPrecision(Desc::mandatory().getInputPrecision());
                 }
             }
+            // for pooling layer output precission is the same as input precission
+            if (LayerInfo(*cnnLayer).isMaxPooling()) {
+                const auto inputPrecission = cnnLayer->insData.front().lock()->getPrecision();
```

Comment:
Typo in inputPrecisssion -> should be inputPrecision. Also comments above.

Reply:
fixed